tsconfig.json

```diff
@@ -35,7 +35,7 @@
       "@grafana/slate-react": ["../node_modules/@types/slate-react"]
     },
     "skipLibCheck": true,
-    "preserveSymlinks": true
+    "preserveSymlinks": false
```

Comment:
Default value is false (as of https://www.typescriptlang.org/docs/handbook/compiler-options.html) so i think we can just remove this line